app/app.global.css

```diff
@@ -89,10 +92,47 @@ input:-webkit-autofill:focus,
 input:-webkit-autofill:active {
   -webkit-box-shadow: 0 0 0 30px white inset !important;
 }
-
 button {
   border: none;
   outline: none;
   font-size: 14px;
   text-transform: uppercase;
 }
+
+/* Custom icons for checkbox and radiobutton */
+.customIcon {
+  height: 16px;
+  width: 16px;
+  background: url('app/components/assets/images/control-icons.svg') no-repeat;
```

Comment:
There are some leftover unused assets in `app/components/assets` - the `app-bg` you have moved under the `Welcome` component, still has an old copy under `app/components/assets/app-bg.png` - please remove such unused assets.